pytest_codestyle.py

```diff
@@ -18,11 +20,14 @@ def pytest_addoption(parser):
                       ignored=pycodestyle.DEFAULT_IGNORE.replace(',', ' ')))
     parser.addini('codestyle_show_source', type="bool", default=True,
                   help='show source code for each error (default: True)')
+    parser.addini('codestyle_exclude', type="pathlist", default=[],
+                  help='source files to exclude from codestyle')
 
 
 def pytest_collect_file(parent, path):
     config = parent.config
-    if config.option.codestyle and path.ext == '.py':
+    print(path)
```

Comment:
please delete this `print`

Reply:
Ah! I knew I wrote that and totally forgot. I'm the worst.